reminder/module.py

```diff
@@ -0,0 +1,465 @@
+
+        try:
+            reaction, user = await self.bot.wait_for(
+                "reaction_add", check=check_id, timeout=60
+            )
+        except asyncio.TimeoutError:
+            await ctx.send(_(ctx, "Reschedule timed out."))
+        else:
+            if str(reaction.emoji) == "✅":
+                await bot_log.debug(
+                    ctx.author,
+                    ctx.channel,
+                    f"Rescheduling reminder - ID: {query.idx}, time: {date}, status: {query.status}",
+                )
+
+                query.reschedule(date)
+                ctx.send(_(ctx, "Reminder rescheduled."))
+            elif str(reaction.emoji) == "❎":
+                ctx.send(_(ctx, "Rescheduling aborted."))
+        try:
+            await message.delete()
+        except (nextcord.errors.NotFound, nextcord.errors.Forbidden):
+            pass
+
+    @commands.guild_only()
+    @commands.check(check.acl)
+    @reminders_.command(pass_context=True, aliases=["remove"])
+    async def delete(self, ctx, idx: int):
+        """Delete reminder"""
+        query = ReminderItem.get_all(guild_id=ctx.guild.id, idx=idx)
+        if query is None:
+            await ctx.send(
+                _(ctx, "Reminder with ID {id} does not exists.").format(id=idx)
+            )
+            return
+
+        query = query[0]
+
+        if query.remind_id != ctx.author.id:
+            await ctx.send(_(ctx, "Can't delete other's reminders."))
+            return
+
+        embed = await self._get_embed(ctx, query)
+        embed.add_field(
+            name=_(ctx, "Do you really want to delete this reminder?"),
+            value=_(ctx, "✅ for confirmation") + _(ctx, "❎ for storno"),
+            inline=False,
+        )
+
+        message = await ctx.send(embed=embed)
+
+        await message.add_reaction("✅")
+        await message.add_reaction("❎")
+
+        def check_id(reaction, user_id):
+            return (
+                reaction.message.id == message.id
+                and (str(reaction.emoji) == "✅" or str(reaction.emoji) == "❎")
+                and user_id.id == query.remind_id
+            )
+
+        try:
+            reaction, user = await self.bot.wait_for(
+                "reaction_add", check=check_id, timeout=60
+            )
+        except asyncio.TimeoutError:
+            await ctx.send(_(ctx, "Deleting timed out."))
+        else:
+            if str(reaction.emoji) == "✅":
+                await bot_log.debug(
+                    ctx.author,
+                    ctx.channel,
+                    f"Deleting reminder from db - ID: {query.idx}, time: {query.remind_date}, status: {query.status}",
+                )
+                query.delete()
+            elif str(reaction.emoji) == "❎":
+                ctx.send(_(ctx, "Deletion aborted."))
+        try:
+            await message.delete()
+        except (nextcord.errors.NotFound, nextcord.errors.Forbidden):
+            pass
+
+    # HELPER FUNCTIONS
```

Comment:
I'd personally move these to the top of the file, so that they are close to the loop where they get called and you don't have to scroll as much while reading them.

For the methods that don't get used by the loop I still believe being at the top of the file is better.

My preffered module layout is:
LOOPS
HELPER methods
COMMANDS
LISTENERS